pkg/systemd/logs.js

```diff
@@ -27,8 +27,6 @@ import ReactDOM from 'react-dom';
 import React from 'react';
 import { EmptyStatePanel } from "cockpit-components-empty-state.jsx";
 
-import "./logs.css";
```

Comment:
Why doesn't this import the .less instead now? logs.html imports logs.css already, but I thought it would be better/more modern to only import CSS in js/webpack instead of on the html page? (To avoid an extra file load and such). It's also inconsistent with the other pages, which do import their *.less.

Reply:
The link tag in the html is the only way to include CSS in our code (afaik).
We could avoid doing it manually and use a plugin for it (html-webpacl-plugin) which basically would create the link elements for us but I think this doesn't need to be done here.
But I don't think there is any way to completely avoid linking the css inside the html, directly or indirectly through some plugin.

Also now it's not inconsistent with other pages, just checked hwinfo, services, I do the same now in all of them.

Reply:
> html-webpacl-plugin

Let's not do this indeed. If we use `<link>`, then we should do it explicitly, otherwise it's too magic and hard to see what's going on -- and it also is harder to ensure a proper ordering of these. Thanks!